frontend/packages/core/src/AppLayout/user.tsx

```diff
@@ -76,34 +109,52 @@ const UserInformation: React.FC = () => {
   }
 
   return (
-    <Grid container alignItems="center" justify="flex-end">
-      <div>{userId()}</div>
+    <Box>
       <UserPhoto
         ref={anchorRef}
         edge="end"
         aria-controls={open ? "account-options" : undefined}
         aria-haspopup="true"
-        onClick={handleToggle}
+        onMouseEnter={handleToggle}
```

Comment:
Did we want this menu to present itself on mouseover? It's a little confusing to have this instead of an explicit click

Reply:
the figma designs mention the avatar popper opens on hover (or maybe I misunderstood "Avatar Hover List" to mean that), so i changed it to onMouseEnter. But I agree, onClick is better. Will change it back